### PR TITLE
Added Module School and new method setup_schools 

### PR DESCRIPTION
**Pull request by Kubasinska**

### CODE REVIEW

File: mesa/Person.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def __init__(self, m:mesaPROTON_OC):
         #print(" ".join(["I am person", str(self.unique_id), "and my model is", str(self.m)]))
 
     def __repr__(self):
-        return "Agent " + str(self.unique_id)
+        return "Agent: " + str(self.unique_id)
```
Comment (mariopaolucci) [inline mesa/Person.py:62]: not here but in a "cleanup" branch..

File: mesa/Person.py [hunk 2]
```diff
@@ -187,24 +187,29 @@ def potential_friends(self):
...
-        self.init_person_empty
-        self.male =  row[1]                            # ...and gender according to values in that row.
-        self.retired = self.age >= m.retirement_age                    # persons older than retirement_age are retired
+    def init_person(self): # person command
+        """
+        This method modifies the attributes of the person instance based on the model's
+        stats_tables as part of the initial setup of the model agents.
+        """
+        row = extra.weighted_one_of(self.m.age_gender_dist, lambda x: x[-1], self.m.rng)  # select a row from our age_gender distribution
+        self.birth_tick =  0 - row[0] * self.m.ticks_per_year      # ...and set age... =
```
Comment (mariopaolucci) [inline mesa/Person.py:196]: extra character at the end of comment?

File: mesa/School.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import extra
+from mesa import Agent, Model
+import mesaPROTON_OC
+import numpy as np
+
+class School():
+    max_id = 0
+    schools = list()
+    def __init__(self, m:mesaPROTON_OC, diploma_level, my_student):
```
Comment (mariopaolucci) [inline mesa/School.py:9]: A school has studentS, why "student"?

Comment (Kubasinska) [reply mesa/School.py:9]: changed into students

**Pull request merged**